workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
Why use the word "tags" at all?  a) It's confusing, since we actually 
have things called "tags", and b) the information isn't actually stored 
as tags: it's either values (in YAML) or meta elements (in the generated 
HTML).